AutomationClient.sc

```diff
@@ -365,7 +365,6 @@ AutomationClient {
     var <>automation = nil,
         <>name = nil,
         <>action = nil,
-        <>minTimeStep = 0.01,
```

Comment:
Ah, thanks for spotting this one as well! What an odd choice by me back then to have a separate time step here.